src/containers/About/index.js

```diff
@@ -4,33 +4,33 @@ import Helmet from 'react-helmet';
 
 export default () => (
   <div>
```

Comment:
Figured we'd try and reference it as `Qur'an` more and more, but it may have been overkill in some places. In particular I think I'll revert the diff right below this comment.